erase_stress_example/main/erase_stress_example.c

```diff
@@ -0,0 +1,150 @@
+/*
+ * Based on [wear levelling example](https://github.com/espressif/esp-idf/tree/master/examples/storage/wear_levelling)
+ */
+#include <stdlib.h>
+#include <stdio.h>
+#include <string.h>
+#include "esp_vfs.h"
+#include "esp_vfs_fat.h"
+#include "esp_system.h"
+#include "esp_random.h"
+
+#include "esp_partition.h"
+#include "esp_flash.h"
+
+#include "esp_log.h"
+
+static const char *TAG = "erase_test_example";
+
+// Mount path for the partition
+const char *base_path = "/spiflash";
+
+// Handle of the wear levelling library instance
+static wl_handle_t s_wl_handle = WL_INVALID_HANDLE;
+
+// whether to only FAT file test or full stress
+#define FAT_ONLY false
+// number of full runs of FAT test + erase stressing
+#define FULL_RUNS 10
+
+void app_main(void)
+{
+    const esp_vfs_fat_mount_config_t mount_config = {
+        .max_files = 4,
+        .format_if_mount_failed = true,
+        .allocation_unit_size = CONFIG_WL_SECTOR_SIZE
+    };
+
+    esp_err_t ret;
+    int full_runs = 0;
+
+start:
```

Comment:
goto not really needed here - see my comments in the simulator code

Reply:
`goto` is no more